arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -0,0 +1,1607 @@
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UserCreateParameters"
+            },
+            "description": "Parameters to create a user."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/tenantIDInPath"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created. Indicates success. The new user is returned in the response body.",
+            "schema": {
+              "$ref": "#/definitions/User"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "User"
+        ],
+        "operationId": "Users_List",
+        "description": "Gets list of users for the current tenant. Reference https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/users-operations#GetUsers",
+        "parameters": [
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/tenantIDInPath"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/UserListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "odata.nextLink",
+          "operationName": "Users_ListNext"
+        },
+        "x-ms-odata": "#/definitions/User"
+      }
+    },
+    "/{tenantID}/users/{upnOrObjectId}": {
+      "get": {
+        "tags": [
+          "User"
+        ],
+        "operationId": "Users_Get",
+        "description": "Gets user information from the directory. Reference: https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/users-operations#GetAUser",
+        "parameters": [
+          {
+            "name": "upnOrObjectId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "User object Id or user principal name to get user information.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/tenantIDInPath"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/User"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "User"
+        ],
+        "operationId": "Users_Update",
+        "description": "Updates an exisitng user. Reference: https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/users-operations#UpdateUser",
```

Comment:
@lmazuel - This will update all the possible properties of an AD user. Password is one of them.
Please take a look at properties of [UserUpdateParameters](https://github.com/Azure/azure-rest-api-specs/pull/344/files#diff-39e2181cd034d406eb4b53cc70670f8cR1496)